tests/test_builder_containerbuild.py

```diff
@@ -114,18 +115,24 @@ def check_meta_entry(filename):
             with open(source_file) as s, open(target_file) as t:
                 assert s.read() == t.read()
 
+        user_warnings = UserWarningsStore()
         log_contents = {}
         for entry in log_entries:
             platform = entry.platform or 'orchestrator'
 
             if platform == builder_containerbuild.METADATA_TAG:
                 check_meta_entry(entry.line)
+            elif user_warnings.is_user_warning(entry.line):
+                user_warnings.store(entry.line)
             else:
                 log_contents[platform] = '{old}{new}\n'.format(
                     old=log_contents.get(platform, ''),
                     new=entry.line
                 )
 
+        if user_warnings:
+            log_contents['user_warnings'] = str(user_warnings)
```

Comment:
List or string?

Reply:
String, because we're saving all user warnings to log as the whole, [see it here](https://github.com/containerbuildsystem/koji-containerbuild/pull/204/files#diff-841526a9ddb532fd3e013f57fbbd6eda075dd0f115db1d4bbf8893b23320a2c2R415)